pkg/omf/functions/cli/omf.cli.install.fish

```diff
@@ -14,8 +14,11 @@ function omf.cli.install
         and require $package
 
       # If package is a theme, set it to active.
-      if contains -- $package (omf.packages.list --theme)
+      set -l themes (omf.packages.list --theme)
+      if contains -- $package $themes
         omf.theme.set $package
+      else if contains -- (omf.packages.name $package) $themes
+        omf.theme.set (omf.packages.name $package)
```

Comment:
This of course calls `omf.packages.name` twice. Which calls `basename` and, because y'all still support fish 2.2.0, `sed`.

There are two alternatives:

1. Store the package name in a variable
2. Use `contains -i`:

```fish
else if set -l ind (contains -i -- (omf.packages.name $package) $themes)
    omf.theme.set $themes[$ind]
```

Reply:
(I make no guarantee that this works with fish 2.2.0, it's been years since I have used that)

Reply:
Works nicely with 2.2.0! Many thanks, @faho 🙌